x-pack/plugins/reporting/server/config/schema.ts

```diff
@@ -6,10 +6,22 @@
  */
 
 import { ByteSizeValue, schema, TypeOf } from '@kbn/config-schema';
+import ipaddr from 'ipaddr.js';
+import { sum } from 'lodash';
 import moment from 'moment';
 
 const KibanaServerSchema = schema.object({
-  hostname: schema.maybe(schema.string({ hostname: true })),
+  hostname: schema.maybe(
+    schema.string({
+      hostname: true,
+      validate(value) {
+        if (ipaddr.isValid(value) && !sum(ipaddr.parse(value).toByteArray())) {
+          // prevent setting a hostname that fails in Chromium on Windows
+          return `cannot use '0.0.0.0' as Kibana host name, consider using the default (localhost) instead`;
```

Comment:
Could we substitute the actual value here? It is going to be a valid IP address, so that it should not print anything insecure.
```suggestion
          return `cannot use '${value}' as Kibana host name, consider using the default (localhost) instead`;
```